xml/System.Runtime.InteropServices/NFloat.xml

```diff
@@ -6834,11 +6843,12 @@ This computes `tan(x * π)`.
         <Parameter Name="result" Type="System.Runtime.InteropServices.NFloat" RefType="out" Index="2" FrameworkAlternate="net-7.0;net-8.0" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="provider">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">The string to parse.</param>
+        <param name="provider">An object that provides culture-specific formatting information about <code data-dev-comment-type="paramref">s</code>.</param>
+        <param name="result">When this method returns, contains the result of successfully parsing <code data-dev-comment-type="paramref">s</code> or an undefined value on failure.</param>
+        <summary>Tries to parse a string into a value.</summary>
+        <returns>
+          <code data-dev-comment-type="langword">true</code> if <code data-dev-comment-type="paramref">s</code> was successfully parsed; otherwise, <code data-dev-comment-type="langword">false</code>.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <paramref name="s" /> was successfully parsed; otherwise, <see langword="false" />.</returns>
```